CHANGELOG.md

```diff
@@ -3,22 +3,34 @@
 All notable changes to this project will be documented in this file.
 
 ## 2.0.0 – 2021-04-07
+
 ### Changed
-- Browserslist config updates, which means some older browsers are no longer supported
-- Dependency updates
+
+-   Browserslist config updates, which means some older browsers are no longer supported
+-   Dependency updates
```

Comment:
Is this intended?

Reply:
Done by prettier to align with tabwidth, lets say you want to break the line then it needs to be indented with tab width, with this change it looks like this:

```
- Before
    Same point
```

```
-   After
    Same point
```